app/user/serializers.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any
+
+from django.contrib.auth import get_user_model
+from django.contrib.sites.shortcuts import get_current_site
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.utils.encoding import force_bytes, force_str
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.validators import UniqueValidator
+
+from app.user.models import Profile
+from app.user.token import account_activation_token
+from speaksfer.settings.base import EMAIL_USER
+
+User = get_user_model()
+
+
+class UserSerializer(serializers.ModelSerializer):
+
+    username = serializers.CharField(
+        max_length=20,
+        min_length=8,
+    )
+
+    email = serializers.EmailField(
+        required=True,
+        validators=[UniqueValidator(queryset=User.objects.all())],
+    )
+
+    password = serializers.CharField(
```

Comment:
Include other password validators, check if it has a number, uppercase, lowercase, symbol

Reply:
The Profile Serializer was created in the first branch we worked on when creating the models